### PR TITLE
Update documentation screenshots

### PR DESCRIPTION
**Pull request by Nate-Wessel**
Updating screenshots in the documentation for the analysis page, along with any text that needs updating to reflect the new layout/settings.

### CODE REVIEW

**Comment by Nate-Wessel** [comment]
Note that I've removed any screenshots in `docs/static/img/` that weren't used/referenced anywhere in the documentation.

File: docs/analysis/configuration.md [hunk 1]
```diff
@@ -8,39 +8,74 @@ Below the accessibility charts, different parameters for the analysis can be set
   <figcaption>The analysis settings pane</figcaption>
 </figure>
 
-The first panel allows the creation and use of **bookmarks**, which store particular analysis settings (e.g. origin location, type of opportunity, departure date and time, travel time cutoff, etc.). Once you have a set of settings you would like save, click <span class="btn btn-success"><i class="fa fa-plus"></i> Create bookmark</span>. Once that is done, you can select a bookmark from the dropdown box to automatically fill in all of the settings from that bookmark. Bookmarks are shared by all projects in a region.
+## Travel time and variability
 
-Next are selectors for **access modes** and **transit modes**; you can choose to perform your analysis with or without transit, and using walking, biking or driving. For instance, in the image above, a combination of walking and transit has been chosen. Note that traffic congestion is not taken into account in driving time estimates, though this may be a feature of a future release when more detailed datasets are available.
+The travel **time cutoff** is the maximum time accepted for a selected portion of trips.
```
Comment (ansoncfit) [inline docs/analysis/configuration.md:13]:
```suggestion
The **Travel time cutoff** sets the maximum door-to-door travel time for the :term:`isochrone` on the map and accessibility indicators.
```

File: docs/analysis/configuration.md [hunk 2]
```diff
@@ -8,39 +8,74 @@ Below the accessibility charts, different parameters for the analysis can be set
...
+## Travel time and variability
 
-Next are selectors for **access modes** and **transit modes**; you can choose to perform your analysis with or without transit, and using walking, biking or driving. For instance, in the image above, a combination of walking and transit has been chosen. Note that traffic congestion is not taken into account in driving time estimates, though this may be a feature of a future release when more detailed datasets are available.
+The travel **time cutoff** is the maximum time accepted for a selected portion of trips. 
+The **Travel time percentile** controls what portion of trips must be at or below this threshold. In single-point analyses the travel time percentile is rounded to one of five pre-defined values (5, 25, 50, 75, and 95). For more information, see [methodology](methodology.html#time-percentile).
```
Comment (ansoncfit) [inline docs/analysis/configuration.md:14]:
```suggestion
There may be variability in travel times, depending on when exactly in a specified time window someone starts their trip. The **Travel time percentile** sets which percentile of the distribution of total travel times to use. Low percentiles correspond to travelers having the information and flexibility to start trips at times that minimize their overall travel time. For more information, see [methodology](methodology.html#time-percentile). In single-point analyses the travel time percentile is rounded to one of five pre-defined values (5, 25, 50, 75, and 95).
```

File: docs/analysis/configuration.md [hunk 3]
```diff
@@ -8,39 +8,74 @@ Below the accessibility charts, different parameters for the analysis can be set
...
 
-The final option is the **Percentile of travel time**. In single-point analyses, this is rounded to one of five pre-defined values (5, 25, 50, 75, and 95). For more information, see [methodology](methodology.html#time-percentile).
+## Access, transit, and egress modes
 
-## Advanced settings
+The three mode settings control what transport modes can be used for travel on the network. Routing can be done using multiple modes, either with or without transit. 
 
-**Maximum transfers** is an upper limit on the number of transfers that will be considered when finding optimal trips.
+The **access mode** controls what mode may be used for the first or only leg of a journey. If any transit modes are enabled, this determines the mode used to access transit from the starting point, or to access the destination directly if that is more efficient than using transit for some part of the journey.
```
Comment (ansoncfit) [inline docs/analysis/configuration.md:24]:
```suggestion
The **access/direct mode** controls what mode may be used for the first or only leg of a journey. If any transit modes are enabled, the selected access mode will be used (up to a maximum time, as described below) to travel from the origin to nearby transit stops, or to access the destination directly if that is faster than using transit. If no transit modes are enabled, the selected mode will be used to travel directly to destinations.
```

File: docs/analysis/configuration.md [hunk 4]
```diff
@@ -8,39 +8,74 @@ Below the accessibility charts, different parameters for the analysis can be set
...
+.. note::
+   Traffic congestion is not taken into account in driving time estimates, though this may be a feature of a future release when more detailed datasets are available.
+
+If one or more of the **transit modes** are enabled, the router will consider journeys using the selected transit modes and will add the option of an egress mode. The default is to use all transit modes, though you may change this for example to examine only a subcomponent of your transit network or to model strong user preferences regarding mode. 
+
+With transit enabled, you will also have to select an **egress mode**, which controls the mode used on the final leg or "last mile" of the trip involving transit. 
+The default egress mode is walking and cycling egress modes may require some extra initial computation time; contact your support team if you need assistance.
+
+Modes can be combined in many different ways, for example:
+* `drive` -> `commuter rail` -> `walk` could be used to simulate suburban park-and-ride commute patterns or
```
Comment (ansoncfit) [inline docs/analysis/configuration.md:35]:
```suggestion
* `drive` -> `commuter rail` -> `walk` could be used to simulate suburban drop-off to downtown commute patterns or
```

Don't want to highlight park-and-ride until it's documented more clearly (which I think @abyrd is working on)

File: docs/analysis/configuration.md [hunk 5]
```diff
@@ -8,39 +8,74 @@ Below the accessibility charts, different parameters for the analysis can be set
...
+
+If your scenario does not include frequency-based routes there is no need to simulate schedules so the requested number of simulated schedules is ignored. In other words, when departure times are explicitly specified for all trips in a scenario, only that single fully specified set of exact departure/arrival times needs to be tested, which speeds computation and eliminates random noise from sampling.
+
+## Routing engine
+
+You will also need to select a **Routing engine** version, which defaults to the highest available version of [Conveyal R5](https://github.com/conveyal/r5). There may be small differences in the results generated by different versions but these should be minimal. When conducting a regional analysis, you'll receive a warning if comparing results from two different versions.
```
Comment (ansoncfit) [inline docs/analysis/configuration.md:63]:
```suggestion
The **Routing engine** version will default to the highest available version of [Conveyal R5](https://github.com/conveyal/r5), or to the version previously used for the selected project. If you use a routing engine version with the first or second number of the version changed, there may be changes to the algorithm that affect results. Previous versions used for regional analysis in a given region will be available in this drop-down menu. You can change versions manually by typing any supported release version (e.g. "v4.8.0") in the routing engine field.
```

Maybe move this to the regional analysis docs:
> When comparing regional analyses run with different routing engine versions, a warning will be displayed.

**Comment by Nate-Wessel** [comment]
@ansoncfit hang on a second before re-reviewing this - there's a screenshot on the regional analysis page I should update as well. Just noticed it.

**Comment by Nate-Wessel** [comment]
OK, @ansoncfit this should be ready for review. There are a few more things I could change on the regional analysis page, but I'm going to try not to rewrite that too much at the moment.

**Pull request merged**